timemachine/cpp/src/kernels/nonbonded_common.cuh

```diff
@@ -0,0 +1,106 @@
+#pragma once
```

Comment:
Since we now have a header and a source file, should we move the implementation over to the source file to have a consistent style?

Reply:
@badisa I might be mistaken but I think the templated functions and those declared with `__forceinline__` need to stay in the header file (templated functions because usage invokes template instantiation and `__forceinline__` so they can be inlined before optimization.